test/options.test.js

```
@@ -1,21 +1,26 @@
+        const rpcs = [{ url: "https://polygon.drpc.org" }, { url: "https://polygon-rpc.com" }];
         const config = await getConfig(
-            rpcs,
-            "0x" + "1".repeat(64), // wallet key
-            "0x" + "3".repeat(40), // arb address
             {
-                lps: ["SUShIswapV2", "bIsWaP"],
+                rpc: rpcs,
+                mnemonic: "test test test test test test test test test test test junk",
+                walletcount: 1,
+                topupAmount: "1",
+                arbAddress: "0x" + "1".repeat(64), // wallet key
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)